### ISSUE TITLE
Sensor selection

### ISSUE THREAD

**Issue by blaqone** [j-kaltes/Juggluco#24]
can you make it possible to select which sensor broadcasts data once there are two/or more sensors connected to juglooco?

At day 13 I insert a new FL3 sensor and let it “warm up” - sometimes the data from the new sensor gets broadcasted. A selection would be awesome!

**Comment by j-kaltes** [comment]
The problem is that you also don't want to use an unreliable sensor for alarms, Libreview and statistics. In that case, you better don't gather data at all from that sensor.  Stop it with Left menu->Sensor->Terminate. You can later reconnect with the sensor by scanning it again.
To see how the sensor is doing, you can connect it with a second exemplar of Juggluco: https://www.juggluco.nl/Juggluco/dub/index.html

**Comment by blaqone** [comment]
This is correct - but wouldn’t it even be better to them both active and see their graphs and decide which one to pick - instead of removing and adding it once in a while to check on its status?

**Comment by j-kaltes** [comment]
Libreview, Statistics, and the Nightscout webserver and uploader also use past values. The program would have to keep track of past sensor exclusions to prevent bad values from being processed when the sensor is selected again for processing. It is much easier to terminate a sensor.

**Comment by blaqone** [comment]
Can you elaborate what you mean she. You speak of „past values“? I think we speak about different things.
